### ISSUE TITLE
Eyelid margin cysts (apocrine hidrocystoma) 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9778]
**Preferred term label:**
Multiple eyelid margin cysts

**Synonyms**
apocrine hidrocystoma

**Definition (free text, please give PubMed ID)**
Presence of many apocrine hidrocystoma (cysts) on the margins of the eyelids.
PMID:19559398 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
224750 

PMID: 19559398  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
